arch/arm/arm_features.h

```diff
@@ -53,4 +53,45 @@ uint32_t crc32_acle(uint32_t crc, const uint8_t *buf, size_t len);
 
 #endif
 
+
+#ifdef DISABLE_RUNTIME_CPU_DETECTION
+//// ARM - SIMD
+#  if (defined(ARM_SIMD) && 0 /* unknown predefined macro */ ) || defined(ARM_NOCHECK_SIMD)
```

Comment:
We must find the macros required for this check (ARM).

Reply:
```__ARM_FEATURE_SIMD32```